erts/test/erlexec_SUITE.erl

```diff
@@ -492,6 +493,18 @@ long_path_env_when_rootdir_not_present(Config) when is_list(Config) ->
     ?assertEqual(string:length(string:find(Output, LongPath ++ pathsep() ++ LongPath)), (LongPathLength * 2) + string:length(pathsep())),
     ok.
 
+otp_9668(_Config) ->
```

Comment:
Can you give the testcase a more descriptive name, like `too_large_argument_test` and then add a comment pointing to the issue/pr.

(The `otp_XYZ` refers to internal ticket numbers, not the github ones so the current name would be very confusing for future maintainers).